app/mailers/patient_mailer.rb

```diff
@@ -207,6 +207,25 @@ def closed_email(patient)
     end
   end
 
+  def closed_sms(patient)
+    if patient&.primary_telephone.blank?
+      add_fail_history_blank_field(patient, 'primary phone number')
+      return
+    end
+    if patient.blocked_sms
+      add_fail_history_sms_blocked(patient)
+      return
+    end
+
+    lang = patient.select_language
+    contents = I18n.t('assessments.sms.closed.dear', locale: lang || :en)
+    contents += "#{patient&.initials_age('-')}.\n"
+    contents += I18n.t('assessments.sms.closed.thank-you', locale: lang || :en)
+
+    # Send message - this needs to be scheduled to send at a reasonable time
+    SendSmsJob.set(wait_until: patient.time_to_contact_next).perform_later(patient.id, contents)
```

Comment:
When is `SendSmsJob` scheduled when `wait_until` occurs in the past? This is the case when it is currently within the contact window.